website/docs/components/models/huggingface.md

```diff
@@ -7,36 +7,48 @@ sidebar_position: 4
 
 To use a model hosted on HuggingFace, specify the `huggingface.co` path in the `from` field and, when needed, the files to include.
 
-### Example: Load a ML model to predict taxi trips outcomes
-```yaml
-models:
-  - from: huggingface:huggingface.co/spiceai/darts:latest
-    name: hf_model
-    files:
-      - path: model.onnx
-    datasets:
-      - taxi_trips
-```
+## Configuration
 
-### Example: Load a LLM model to generate text
-```yaml
-models:
-  - from: huggingface:huggingface.co/microsoft/Phi-3.5-mini-instruct
-    name: phi
-```
+### `from`
 
-### Example: Load a private model
-```yaml
-models:
-  - name: llama_3.2_1B
-    from: huggingface:huggingface.co/meta-llama/Llama-3.2-1B
-    params:
-      hf_token: ${ secrets:HF_TOKEN }
+The `from` key takes the form of `huggingface:model_path`, which follows the following regex format.
+
+```regex
```

Comment:
It seems to be very complex for the first thing on the page, I would simplify this to examples below that will cover 95% of all user cases IMO and add additional section separatly after main details

```
- `huggingface:username/modelname`: Implies the latest version of `modelname` hosted by `username`.
- `huggingface:huggingface.co/username/modelname:revision`: Specifies a particular `revision` of `modelname` by `username`, including the optional domain.
- ``